server/datastore/mysql/software.go

```diff
@@ -2524,6 +2546,8 @@ INNER JOIN software_cve scve ON scve.software_id = s.id
 			st.id as software_title_id,
 			s.id as software_id,
 			s.version,
+			s.bundle_identifier,
```

Comment:
Guessing adding the bundle ID is for the same FR? I see why you added source.

Reply:
Yes, we need both `bundle_identifier` and `team_identifier` on the hosts/:id/software endpoint.